Date/Human.php

```diff
@@ -197,7 +197,7 @@ function gregorianToHuman($day = 0, $month = 0, $year = 0)
      * @access   public
      * @static
      */
-    function humanToGregorian($day, $month, $year = 0)
+    static function humanToGregorian($day, $month, $year = 0)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.